### ISSUE TITLE
[Linux]  dark mode switch follow gtk theme

### ISSUE THREAD

**Issue by icepie** [Alex313031/Thorium#74]
**Is your feature request related to a problem? Please describe.**
I found a [patch](https://github.com/qcasey/chromium-patches#gtk-dark-theme) for implement the feat 

**Additional Notes**
[patch](https://github.com/qcasey/chromium-patches#gtk-dark-theme)

**Comment by Alex313031** [comment]
@icepie Good idea! I will add this to Thorium, and credit you in the copyright header of the modified files

**Comment by Alex313031** [comment]
@icepie Here you go > and you can see i credited the original author and you > https://github.com/Alex313031/Thorium/blob/main/src/ui/gtk/native_theme_gtk.cc#L1

**Comment by icepie** [comment]
wow! thanks my bro!

**Comment by Alex313031** [comment]
@icepie Your welcome. Lol this is why i made the new issues templates, to encourage more things like this where people can actively improve thorium without having to actually modify code and push to the repo.

**Comment by Alex313031** [comment]
@icepie https://github.com/Alex313031/Thorium/releases/tag/M107.0.5280.0

**Comment by raisedintheusa** [comment]
I'm not the biggest fan of this change. There's no way to disable this, right? There already is a flag to force dark mode if desired. I like a dark GTK theme for other applications with a more consistent design. And I like having the tab strip/address bar dark on Thorium. But websites are all so different that a global dark mode has a very inconsistent effect. Some sites are not made dark so I still require Dark Reader for them. Other sites become ugly or hard-to-read when made dark and there might be no option on the site to change it. Perhaps there could be a switch to disable this feature.

**Comment by Alex313031** [comment]
@raisedintheusa I can probably set it behind a chrome://flag and name it something like GTK Auto-Dark Mode
.

**Comment by Alex313031** [comment]
@icepie @raisedintheusa These patches should make a flag to enable/disable the feature. > https://github.com/Alex313031/Thorium/commit/29e1d91afc5dc349bcf92c03f9b01b456a7a5be1  &  https://github.com/Alex313031/Thorium/commit/d5fe7f8e0049058b4bb0ffc59fca0d2b7d510a99

**Comment by icepie** [comment]
> @icepie @raisedintheusa These patches should make a flag to enable/disable the feature. > [29e1d91](https://github.com/Alex313031/Thorium/commit/29e1d91afc5dc349bcf92c03f9b01b456a7a5be1) & [d5fe7f8](https://github.com/Alex313031/Thorium/commit/d5fe7f8e0049058b4bb0ffc59fca0d2b7d510a99)

good work！

**Comment by Alex313031** [comment]
@icepie @raisedintheusa @tomByrer Latest release has this patch. It will be turned off by default (using the old behavior where one can manually control dark mode with the chrome://flags#force-dark-mode flag), but you can turn this flag on to allow auto gtk dark mode setting > chrome://flags#auto-dark-mode

**Comment by raisedintheusa** [comment]
> Latest release has this patch.
Thanks, works good!

**Comment by Alex313031** [comment]
@raisedintheusa @icepie Closing as completed.

**Comment by WingofaGriffin** [comment]
Is it possible to add these flags as sliders in the settings page? Mercury has the Firefox theme selector, and I think something similar in settings would be a nice feature for a more obvious dark mode.

**Comment by Alex313031** [comment]
@WingofaGriffin For a more obvious dark mode, disable this flag, and enable the chrome://flags/#force-dark-mode flag.

Also, If you are on linux, you will probably also want to set chrome://settings/appearance Theme to "Classic" instead of "GTK"

**Comment by WingofaGriffin** [comment]
Apologies, I wasn't clear in my first post.

I already found both of these flags on my own, but I think it would be smart to make them a part of the settings GUI or the extension for easier access for users who are afraid/don't know about the flags.
